src/pages/Dashboard/Dashboard.tsx

```diff
@@ -309,281 +310,287 @@ export default function Dashboard(): JSX.Element {
+                                icon={faPlay as IconProp}
+                              />
+                            </button>
+                          </Col>
+                          <Col className={styles.toolbarColumn} sm="1">
+                            <button
+                              className={styles.toolbarButton}
+                              onClick={handleOpenCommitModal}
+                              id="CommitButton"
+                            >
+                              <FontAwesomeIcon
+                                title="Commit"
+                                icon={faCodeBranch as IconProp}
+                              />{' '}
+                            </button>
+                          </Col>
+                        </>
+                      ) : (
+                        <></>
+                      )}
+                      <Col className={styles.toolbarColumn} sm="1">
+                        <button
+                          className={styles.toolbarButton}
+                          onClick={handleSubmit}
+                          ref={submitButtonRef}
+                          id="SubmitButton"
+                        >
+                          <FontAwesomeIcon
+                            title="Submit"
+                            icon={faCloudUploadAlt as IconProp}
+                          />{' '}
+                        </button>
+                      </Col>
+                    </div>
+                  </div>
+                  <div>
+                    <div className={styles.settingsIconDiv}>
+                      <div className={styles.settingsIcon} id="Settings">
+                        <FontAwesomeIcon
+                          title={'Settings'}
+                          icon={faGear as IconProp}
+                          color={'#cbcbcb'}
+                          onClick={handleOpenSettings}
+                          className={styles.hoverIcon}
+                        />
+                      </div>
+                      <div className={styles.settingsIcon} id="Shortcuts">
+                        <FontAwesomeIcon
+                          title={'Shortcuts'}
+                          icon={faCircleInfo as IconProp}
+                          color={'#cbcbcb'}
+                          onClick={handleOpenInfo}
+                          className={styles.hoverIcon}
+                        />
+                      </div>
+                    </div>
+                  </div>
+                  <div>
+                    <Button
+                      className={styles.closeEditorButton}
+                      onClick={() => {
+                        updateDividerPosition(dividerPosition - 1);
+                      }}
+                      variant="dark"
+                    >
+                      <FontAwesomeIcon
+                        size={'sm'}
+                        icon={faChevronLeft as IconProp}
```

Comment:
you can remove the common buttons outside the if statement here, will save on code

Reply:
here It cant be made common 
because of react bootstarp buutton toolbar.
Thats why Shubam made it seperate.